src/config.go

```diff
@@ -34,9 +26,24 @@ func NewConfigFile() (*Config, error) {
 		return nil, fmt.Errorf("Error parsing config file: %s", err)
 	}
 
+	if isDockerComposeVersion2(c.Config) {
+		services, ok := getMapVal(c.Config, "services").(yamlConfig)
+		if ok {
+			c.Config = services
+		}
+	}
+
```

Comment:
Added this block of code to decide if the docker-compose file uses version 2, and if it does, reassign everything under `services` to `Config.Config`.